src/client.cpp

```diff
@@ -64,6 +67,7 @@ CClient::CClient ( const quint16  iPortNumber,
     bFraSiFactDefSupported           ( false ),
     bFraSiFactSafeSupported          ( false ),
     eGUIDesign                       ( GD_ORIGINAL ),
+    bDisplayChannelLevels            ( false ),
```

Comment:
I would enable the new functionality by default, i.e. for a new user which had not started Jamulus before, the channel meters should appear. So I would set there here to "true" instead of "false".

Reply:
OK - and yes, I'll get onto the manual changes, too.

Reply:
Sorry for giving you a hard time. But you should know me from previous code deliveries so you knew what to expect ;-).

Reply:
Oh, you're no more vicious than I am when reviewing code :).